_resources/port1.0/group/java-1.0.tcl

```diff
@@ -58,14 +47,19 @@ proc find_java_home {} {
 
     global java.version java.fallback
     if { ${java.version} ne "" } {
-        if { [catch {set val [exec "/usr/libexec/java_home" "-f" "-v" ${java.version}]}] } {
+        set f_param "-f"
+        global os.platform os.major
+        if {${os.platform} eq "darwin" && ${os.major} == 20} {
```

Comment:
Probably better to make that check darwin >= 20 to cover future versions, intil we know when/if apple fixes the bug.